packages/edit-post/src/components/sidebar/post-status/index.js

```diff
@@ -52,7 +57,7 @@ export default function PostStatus() {
 				isOpened: isEditorPanelOpened( PANEL_NAME ),
 				// Post excerpt panel is rendered in different place depending on the post type.
 				// So we cannot make this check inside the PostExcerpt component based on the current edited entity.
-				showPostExcerptPanel: ! [
+				showExcerptAndLastEditedPanels: ! [
```

Comment:
It seems the condition's name is getting longer and longer 😄

Should we use something more general - `showNormalPostPanels` or `showPostContentPanels`?